schemas/cslc_s1.yaml

```diff
@@ -0,0 +1,74 @@
+runconfig:
+    name: str()
+
+    groups:
+        pge_name_group:
+           pge_name: enum('CSLC_S1_PGE')
+
+        input_file_group:
+           # Required. List of SAFE files (min=1)
+           safe_file_path: list(str(), min=1)
+           # Required. List of orbit (EOF) files
+           orbit_file_path: list(str(), min=1)
+           # List of (unique) burst ID to process
+           burst_id: list(str(), min=0, max=27, required=False)
```

Comment:
`max=27` is for all bursts in a swath? Are there cases where there's more than 9 bursts per sub-swath? Will other Sentinel 1 modes (besides IW) be used for CSLC? If so does the cap make work for those modes?

Reply:
Good point. To my knowledge, typically each sub-swath has 9 bursts. However, I was not able to confirm that number consulting official Sentinel pages. I am inclined to remove the cap on the max number of bursts to process